src/skb_font_collection.c

```diff
@@ -369,16 +368,25 @@ skb_font_handle_t skb_font_collection_add_font(skb_font_collection_t* font_colle
 	assert(font_idx != SKB_INVALID_INDEX);
 
 	skb_font_t* font = &font_collection->fonts[font_idx];
+	memset(font, 0, sizeof(skb_font_t));
```

Comment:
Moved the clear to this point, as this is the place where we know both the handle and the generation.

Reply:
Splitting hair... this functions starts to look more lie `skb__alloc_font()` than allocating a handle. I'm fine either way.